src/analysis/pluginDeclaration.js

```diff
@@ -25,3 +27,15 @@ export function combineTypes(
   const edgeTypes = [].concat(...decs.map((x) => x.edgeTypes));
   return {nodeTypes, edgeTypes};
 }
+
+export function weightsForDeclaration(dec: PluginDeclaration): WeightsT {
```

Comment:
Nit: slight preference for `declaration` over `dec`.

Reply:
Ack(nowledged); not making changes.